eth/fetcher/fetcher_test.go

```diff
@@ -296,6 +301,15 @@ func verifyImportDone(t *testing.T, imported chan *types.Block) {
 	}
 }
 
+func verifyProposeBlockHandlerCalled(t *testing.T, proposedBlockChan chan *types.Header) bool {
+	select {
+	case <-proposedBlockChan:
+		return true
+	case <-time.After(50 * time.Millisecond):
+		return false
```

Comment:
t.Fatalf("did not call propose block handler")

Reply:
what's the difference to return false and asserted at the test function?

Reply:
return false doesn't fail test, t.Fatal will fail test

Reply:
updated already